scripts/travis.sh

```diff
@@ -5,8 +5,11 @@ SUBFOLDER=$1
 
 function test {
     set -ev
+    npm install
+    npm run build
     cd $TRAVIS_BUILD_DIR/$SUBFOLDER
     npm install
+    pwd
```

Comment:
Guess that was for testing, but isn't necessary anymore? 😉

Reply:
yeah stuff is lil bit messed up right now